.fixtures.yml

```diff
@@ -1,7 +1,7 @@
 fixtures:
-  forge_modules:
-    concat: puppetlabs-concat
-    stdlib: puppetlabs-stdlib
-    selinux: puppet-selinux
+  repositories:
+    concat: 'git://github.com/puppetlabs/puppetlabs-concat'
```

Comment:
I'd prefer https since it's secure (git:// is plaintext) and in environments with strict firewalls it tends to work as well.